### ISSUE TITLE
Discussion

### ISSUE THREAD

**Issue by wahasa** [wahasa/Ubuntu#3]

**Comment by wahasa** [comment]
after installing ubuntu did you install desktop. If it is already

basic vnc server commands

* start vnc :
vncserver -geometry 1600x900 -remote desktop name :1

* stop vnc :
vncserver -kill :1

**Comment by Alex460k** [comment]
Yes. I figured it out myself bit it was a problem. So i inatalled ubuntu,
idk what version 22.04 or 23.04 and then i installed kde desktop, but when
i was typing ,, vnc-start " it says ,, command not found " so i wrote this
issue on your github page and then i decided to reinstall kde. After that
it showed me the vnc server but it had a problem. So i uninstalled it.

**Comment by Alex460k** [comment]
> after installing ubuntu did you install desktop. If it is already
> 
> basic vnc server commands
> 
> * start vnc :
>   vncserver -geometry 1600x900 -remote desktop name :1
> * stop vnc :
>   vncserver -kill :1

Can you tell me the full commands? Like from nano and stuff. Like every single step. Please
